src/carousel/carousel.ts

```diff
@@ -184,7 +184,7 @@ export class NgbCarousel implements AfterContentChecked,
       this._ngZone.runOutsideAngular(() => {
         const hasNextSlide$ = combineLatest([
                                 this.slide.pipe(map(slideEvent => slideEvent.current), startWith(this.activeId)),
-                                this._wrap$, this.slides.changes.pipe(startWith(<{}>null))
+                                this._wrap$, this.slides.changes.pipe(startWith(<null>null))
```

Comment:
Is it necessary to cast here ? I noticed that removing the type (just `startWith(null)`compiles fine).

Reply:
Indeed, I haven't checked it it compiles without casting. There were lining issues related to `rxjs` version before this. Will remove casting.